src/core_plugins/kibana/public/visualize/wizard/wizard.js

```diff
@@ -43,10 +52,17 @@ routes.when('/visualize/step/2', {
   }
 });
 
-module.controller('VisualizeWizardStep2', function ($route, $scope, $location, timefilter, kbnUrl) {
+module.controller('VisualizeWizardStep2', function ($route, $scope, timefilter, kbnUrl) {
   const type = $route.current.params.type;
+  $scope.addToDashMode = $route.current.params[DashboardConsts.ADD_TO_DASH_PARAM];
```

Comment:
If the template doesn't need access to addToDashMode then this should just be a const instead of a scope variable.